spec/views/complaints/_pagination.erb_spec.rb

```diff
@@ -22,7 +22,7 @@
     it "has no pagination" do
       @pagy = Pagy.new(count: 25)
 
-      render partial: "complaints/pagination", locals: {pagy: @pagy}
+      render partial: "complaints/pagination", locals: {pagy: @pagy, path_args: {}}
```

Comment:
[suggestion] I realize it's being tested above, but I don't hate the idea of explicitly testing that arguments are being treated correctly by this partial, too.  Thoughts?

Reply:
Hmm, I'll add checks for previous and next, since they're generated in the partial and not the helper.